src/media/js/addon/reducers/addon.js

```diff
@@ -73,6 +73,17 @@ export default function addonReducer(state=initialState, action) {
       return newState;
     }
 
+    case dashboardActions.DELETE_OK: {
+      /*
+        Mark add-on as deleted.
+
+        payload (string) -- addonSlug
+      */
+      const newState = _.cloneDeep(state);
+      newState.addons[action.payload].deleted = true;
```

Comment:
I guess that's true. We'll have to filter on this property in listing pages, though.